src/Stage/Emit/JavaScript.elm

```diff
@@ -135,6 +135,16 @@ emitExpr located =
         Tuple3 e1 e2 e3 ->
             "[" ++ emitExpr e1 ++ "," ++ emitExpr e2 ++ "," ++ emitExpr e3 ++ "]"
 
+        Record bindings ->
+            let
+                bindingsJS =
+                    bindings
+                        |> Dict.values
+                        |> List.map (\binding -> mangleVarName binding.name ++ " = " ++ emitExpr binding.body)
+                        |> String.join ", "
```

Comment:
I'm pretty sure this is not JavaScript syntax :D

Reply:
Oh my! Too much elm in my life!

Reply:
You're saying that like it's a bad thing :stuck_out_tongue: